.gitignore

```diff
@@ -0,0 +1,53 @@
+# Byte-compiled / optimized / DLL files
+ __pycache__/
+ *.py[cod]
+ *$py.class
+
+# C extensions
+ *.so
+
+ # Installer logs
+ pip-log.txt
+ pip-delete-this-directory.txt
+
+ # Unit test / coverage reports
+ htmlcov/
+ .tox/
+ .coverage
+ .coverage.*
+ .cache
+ nosetests.xml
+ coverage.xml
+ *.cover
+ .hypothesis/
+ .pytest_cache/
+
+ # pyenv
+ .python-version
+
+ # Environments
+ .env
+ .envrc
+ .venv
+ env/
+ venv/
+ ENV/
+ env.bak/
+ venv.bak/
+
+ # mypy
+ .mypy_cache/
+
+ # Emacs
+ \#*
+ *~
+
+ # Other editors
+ .idea
+ .vscode
+
+ # Meta files
+ *.DS_Store
+
+# deployment / artifacts
+.serverless/
```

Comment:
Should we not be adding .idea to the git ignore? Not sure we want these up on git but need to confirm.

Reply:
Dunno

Reply:
Ok, will confirm with @bitmonkey

Reply:
test